### ISSUE TITLE
Doc Errors related to  tf.space_to_batch

### ISSUE THREAD

**Issue by asetsuna** [tensorflow/tensorflow#5594]
In the[ api-doc related to](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/array_ops.md#tfspace_to_batchinput-paddings-block_size-namenone-space_to_batch) `tf.space_to_batch`, the result of example case of 

> (3) For the following input of shape `[1, 4, 4, 1]` and block_size of 2:
```prettyprint
x = [[[[1],   [2],  [3],  [4]],
      [[5],   [6],  [7],  [8]],
      [[9],  [10], [11],  [12]],
      [[13], [14], [15],  [16]]]]
```

should be 
```prettyprint
x = [[[[1], [3]], [[9], [11]]],
     [[[2], [4]], [[10], [12]]],
     [[[5], [7]], [[13], [15]]],
     [[[6], [8]], [[14], [16]]]]
```
rather than 
```prettyprint
x = [[[[1], [3]], [[5], [7]]],
     [[[2], [4]], [[10], [12]]],
     [[[5], [7]], [[13], [15]]],
     [[[6], [8]], [[14], [16]]]]
```

Docs related to `tf.space_to_batch_nd`, `tf.batch_to_space_nd`, `tf.batch_to_space` have similar problems in their **(3)** cases, too.

**Comment by girving** [comment]
@gpapan Does this look right to you?

**Comment by aselle** [comment]
The new  doc link is here, and it does seem to have  the problem still and prettyprint shown in all examples.
https://www.tensorflow.org/api_docs/python/tf/space_to_batch

**Comment by wolffg** [comment]
This is fixed in master:  
https://www.tensorflow.org/versions/master/api_docs/python/tf/space_to_batch 

and in v1.2:
https://www.tensorflow.org/versions/r1.2/api_docs/python/tf/space_to_batch

The quoted link will get fixed automatically when we switch v1.2.  We generally don't cherrypick back cosmetic change like this, so r1.1 will have this for the foreseeable future.  I think we can close this.
